### PR TITLE
EPMRPP-36690 React. Widget creation request

### PR DESCRIPTION
**Pull request by AmsterGet**
ToDo was created for WS.

### CODE REVIEW

**Comment by AmsterGet** [comment]
New widget configuration template - https://pastebin.com/MZiYgbKH
ToDo was created for WS.

File: app/src/pages/inside/dashboardItemPage/widgetWizardModal/widgetControls/constants.js [hunk 1]
```diff
@@ -1,3 +1,26 @@
+import PropTypes from 'prop-types';
+import {
```
Comment (DiscoElevator) [inline app/src/pages/inside/dashboardItemPage/widgetWizardModal/widgetControls/constants.js]: `import * as WIDGET_TYPES from 'common/constants/widgetTypes';`

File: app/src/pages/inside/dashboardItemPage/widgetWizardModal/widgetControls/constants.js [hunk 2]
```diff
@@ -53,3 +76,37 @@ export const CONTENT_FIELDS = {
   LOGGED_OBJECT_REF: 'loggedObjectRef',
   HISTORY: 'history',
 };
+
+export const requestDataShape = PropTypes.shape({
+  filterId: PropTypes.array,
+  description: PropTypes.string,
+  name: PropTypes.string,
+  share: PropTypes.bool,
+  widgetType: PropTypes.oneOf([
```
Comment (DiscoElevator) [inline app/src/pages/inside/dashboardItemPage/widgetWizardModal/widgetControls/constants.js]: `widgetType: PropTypes.oneOf(Object.keys(WIDGET_TYPES).map((key) => WIDGET_TYPES[key]))`

**Comment by AmsterGet** [comment]
Can't to reproduce the `error` linke in the summary

File: app/src/pages/inside/dashboardItemPage/widgetWizardModal/widgetControls/constants.js (hunk 1 above)
Comment (AmsterGet) [reply app/src/pages/inside/dashboardItemPage/widgetWizardModal/widgetControls/constants.js]: Done

File: app/src/pages/inside/dashboardItemPage/widgetWizardModal/widgetControls/constants.js (hunk 2 above)
Comment (AmsterGet) [reply app/src/pages/inside/dashboardItemPage/widgetWizardModal/widgetControls/constants.js]: Fixed

**Pull request merged**